### ISSUE TITLE
Remove event tile on main screen

### ISSUE THREAD

**Issue by dsarkar** [corona-warn-app/cwa-wishlist#469]
## Current Implementation

Check-In related functionalities are not bundled in one single place:
- As a host I have to use the main screen: the event tile
- As a attendee I have to use the Check-In tab

## Suggested Enhancement

- Bundle all check-in related functionalities at a single point.
- With the current design that would be: access all check-in related functions vie the check-in tab.

## Expected Benefits

- Improve UX
- Clearer design of the app
- Uncluttered main screen
- improve feel and look of an app, main screen feels like a website

**Comment by MikeMcC399** [comment]
Also, for creators of QR codes, the only way to view an existing QR code is to click on the button "CREATE QR CODE". That doesn't feel right that I have to tap "CREATE" to in order to "VIEW".

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks. I will add it to the OP, if you don't mind.

**Comment by MikeMcC399** [comment]
@dsarkar 
> I will add it to the OP, if you don't mind.

Certainly! Maybe a more general label like "Manage QR codes" would be better than "Create QR code". The text above would need changing as well.

**Comment by achisto** [comment]
Great request. I was really confused when I first used CWA today because I couldn't find the "create QR code" functionality. In the end I only found it after looking at screenshots because I would have never guessed to find it on the main screen.

**Comment by fynngodau** [comment]
The current design is indeed a bit hard to understand.

A user correctly noted that it is confusing that the tab with the QR code icon does not allow one to create QR codes.

If no better solution is found, there could be a link between <del>worlds</del> the attendee and the organizer view through the overflow drop-down menu.

**Comment by nilsriesenberg** [comment]
Add a button to create a QR-Check-In code in the "Check-In" submenu.

## Current Implementation

Currently the button to create a QR code only exists on the main page and the users are unable to create a QR code from the "Check-In" submenu.
This is against the expected behaviour of the application and leads to confusion.

## Suggested Enhancement

Add a button to create a code in the "Check-In" page.
Or add an entry in the submenu of the "Check-In" page to generate a new event code.

## Expected Benefits

better UX, closer to an expected behaviour of the app

**Comment by dsarkar** [comment]
@nilsriesenberg  Thanks. Added your suggestion to Internal Tracking ID: EXPOSUREAPP-6617

**Comment by Ein-Tim** [comment]
One thing that I want to add here: Inform the user that you have moved the feature to another place to avoid confusing similar to what happened when moving the certificates to it's own tab (:

**Comment by Ein-Tim** [comment]
The situation @MikeMcC399 mentioned in https://github.com/corona-warn-app/cwa-wishlist/issues/469#issuecomment-824034396 got worse with the introduction of the "Warn for others" feature. 
You have to click "Create QR code" to use it. 
I meanwhile rather consider this as an (easy to fix (text change only)) bug, rather than a enhancement request, what about you @MikeMcC399?
